lib/screens/login_screen.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:dit_events/custom_widgets/divider.dart';
+import 'package:dit_events/custom_widgets/guest.dart';
+import 'package:flutter/material.dart';
+
+import '../custom_widgets/inputfield_widget.dart';
+import '../custom_widgets/loginbtn.dart';
+import '../custom_widgets/text_widget.dart';
+import '../custom_widgets/guest.dart';
+class Login_screen extends StatefulWidget {
+  const Login_screen({super.key});
+  static const routeName = '/login-screen';
+  @override
+  State<Login_screen> createState() => _Login_screenState();
+}
+
+class _Login_screenState extends State<Login_screen> {
+  @override
+  Widget build(BuildContext context)
+  {
+    return Scaffold(
+      body:  Center(
+
+        child: Column(
```

Comment:
add margin to column instead of container remove margin from inner child container.